chapter07/qli/ex4.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+while( <> ){
+  if( /[A-Z][a-z]/ ){
```

Comment:
While this catches the obvious what about cases where there are capitals at the end of the word not the beginning or other characters i.e. the A-team.